dds/DCPS/RTPS/ParameterListConverter.cpp

```diff
@@ -1059,9 +956,15 @@ int from_param_list(const ParameterList& param_list,
         writer_data.ddsPublicationData.reliability = param.reliability();
         // Interoperability note:
         // Spec creators for RTPS have reliability indexed at 1
-        writer_data.ddsPublicationData.reliability.kind =
-          (DDS::ReliabilityQosPolicyKind)
-              ((int)writer_data.ddsPublicationData.reliability.kind - 1);
+        {
+          const CORBA::Short rtpsKind = (const CORBA::Short)(param.reliability().kind);
+          if(rtpsKind == RTPS::BEST_EFFORT){
```

Comment:
Ditto above... fix the whitespace / braces to match the guidelines. Two more instances below.